cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/service/checker/AbstractGcpBaseResourceChecker.java

```diff
@@ -46,7 +46,7 @@ protected void exceptionHandler(GoogleJsonResponseException ex, String name, Res
         }
     }
 
-    protected GcpResourceException exceptionHandlerWithThrow(GoogleJsonResponseException ex, String name, ResourceType resourceType) {
+    protected static GcpResourceException exceptionHandlerWithThrow(GoogleJsonResponseException ex, String name, ResourceType resourceType) {
```

Comment:
Why were these methods made static? I don't see them referenced in a static context anywhere in this review.

Reply:
I had made these static to be referenced staticly from the new Reserved IP resource builder, but then I realized it would be just as easy to inject the old one. Thanks for bringing this up.